tests/js/realm-tests.js

```diff
@@ -2033,56 +2033,6 @@ module.exports = {
     realm.close();
   },
 
-  testWriteCopyTo: function () {
```

Comment:
Should this really be removed in its entirety? That leaves the feature completely untested, doesn't it?

Reply:
It's tested twice, so I removed it from there.

Reply:
Ahh okay 👍